scripts/happy

```diff
@@ -1164,6 +1164,10 @@ def push(ctx, images, tag, extra_tag, compose_env):
     # images to the right places with some re-tagging.
     first_registry = get_ecr_repo(config)
     builder_config["ecr_repo"] = first_registry  # TODO this is a leak in our abstraction, need to fix.
+    # HACK: Sets happy env name to the first extra tag; we need to know the happy env name at docker image build time, unfortunately, since it's not parameterized in the Docker image env
+    rdev_stack_name = extra_tag[0]
```

Comment:
@atolopko-czi it seems like this breaks running `./scripts/happy create RDEV_NAME` in Data Portal repo directly for some reason 😄 I'm currently just hacking it by hard coding `rdev_stack_name = "thuang-wmg-2"`, so I can push to rdev. So not a blocker! PTAL tomorrow thank you 🙏

Reply:
Thanks for the bug report!  [Will investigate](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1682).